### ISSUE TITLE
[Shattered Halls Bugs - Grand Master Nethekurse - abuses and bugs]

### ISSUE THREAD

**Issue by Paradigmxd** [chromiecraft/chromiecraft#5690]
### What client do you play on?

enUS

### Faction

Both

### Content Phase:

70

### Current Behaviour

Where do we start ...

1. Pet bug can be used to send through the door and pull him out of the room which renders the Sewage event useless
2. If Players go through sewage event and do the boss traditional way - The Boss paths back and forth on his platform ALL THE WHILE WHILE YOU ARE FIGHTING HIM....

### Expected Blizzlike Behaviour

1. Pet bugs should not exist - Blizzard created the Sewage event for a reason so players can go through it 
2. Sewage event doesn't work - meaning there is a timer for the mobs when killed they respawn...
3. The boss does not just path back and forth while engaged.. (he does that before engage...)

### Source

https://www.youtube.com/watch?v=gAhZe5QEgLc

### Steps to reproduce the problem

Fight Grand Master Nethekurse in Shattered Halls
Go through sewage event 
use pet to pull him through the door

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/bc0ddaa3c2af2e7b0e92986741485d04c1a90a35

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by balleny** [comment]
> Pet bug can be used to send through the door and pull him out of the room which renders the Sewage event useless

at some point in retail, it was possible to skip sewage event with relog / pet iirc.

> If Players go through sewage event and do the boss traditional way - The Boss paths back and forth on his platform ALL THE WHILE WHILE YOU ARE FIGHTING HIM....

see also https://github.com/chromiecraft/chromiecraft/issues/5681

**Comment by Paradigmxd** [comment]
Yes - I also said that during our last run - but during TBC classic this was no longer possible

**Comment by Tereneckla** [comment]
Thank you for the report, these issues have been addressed already
